simulariumio/smoldyn/smoldyn_data.py

```diff
@@ -71,3 +76,53 @@ def __init__(
             spatial_units if spatial_units is not None else UnitData("m")
         )
         self.plots = plots if plots is not None else []
+
+    @classmethod
+    def from_dict(
+        cls, buffer_data: Dict[str, Any]
+    ):
+        """
+        Create SmoldynData from a simularium JSON dict containing buffers
+
+        Parameters
+        ----------
+        buffer_data: Dict[str, Any]
+            JSON dict containing values key value pairs representing the
+            data to be turned into a SmoldynData object
+        """
+        display_data = None
```

Comment:
this chunk processing display data could be broken into a function. Likely we'll want to use it for other converters too, so maybe in utils

Reply:
sounds like a good idea! my quick attempt at doing this just failed, because of where the util file is located and where it's called, I was creating a circular dependency. gotta take a moment to actually plan how the util file should be organized